apps/emails-and-messages/src/modules/event-handlers/message-event-types.ts

```diff
@@ -0,0 +1,19 @@
+export const messageEventTypes = [
```

Comment:
you can type it `messageEventTypes: Array<AsyncWebhookEventType>  = []` to avoid typos (AsyncWebhookEventType is from SDK)

Reply:
Will you add it?